array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(num => num * 2);
+console.log('doubled:', doubled);
+
+const prices = numbers.map(num => `$${num.toString()}.00`);
```

Comment:
There is no need to use `toString` here:
```JAVASCRIPT
`$${num}.00`
```
would work the same.

Also we don't want to hard code the 2 spots after the decimal, what if the array had numbers with decimals? Look into using the `toFixed` method.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed